src/components/TasksTable/index.jsx

```diff
@@ -335,47 +413,57 @@ export default class TasksTable extends Component {
             renderRow={item => (
-              <TableRow tabIndex={-1} key={item.summary}>
-                <TableCell
-                  component="th"
-                  scope="row"
-                  className={classes.tableCell}>
-                  {item.project}
-                </TableCell>
-                <TableCell className={classes.tableCell}>
-                  <IconButton
-                    name={item.summary}
-                    className={classes.infoButton}
-                    onClick={this.handleDrawerOpen}>
-                    <InformationVariantIcon />
-                  </IconButton>
-                  <List dense className={classes.summary}>
-                    <ListItem
-                      classes={{
-                        gutters: classes.summaryItem,
-                      }}
-                      title={item.summary}
-                      button
-                      target="_blank"
-                      rel="noopener noreferrer"
-                      component="a"
-                      href={item.url}>
-                      <ListItemText
-                        primary={
-                          <div className={classes.summaryText}>
-                            {item.summary}
-                          </div>
-                        }
-                      />
-                      <LinkIcon className={classes.icon} size={iconSize} />
-                    </ListItem>
-                  </List>
-                </TableCell>
-                <TableCell className={classes.tableCell}>
+              <div>
+                <List
+                  disablePadding
+                  dense={this.props.width === 'xs'}
+                  className={classNames(classes.summary, classes.cardTitle)}>
+                  <ListItem
+                    classes={
+                      (classes.cardItem,
+                      {
+                        gutters: classes.cardSummaryItem,
+                      })
+                    }
+                    title={item.summary}
+                    button
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    component="a"
+                    href={item.url}>
+                    <ListItemText
+                      primary={
+                        <div className={classes.summaryText}>
+                          {`${item.project} - ${item.summary}`}
+                        </div>
+                      }
+                    />
+                    <LinkIcon size={iconSize} />
+                  </ListItem>
+                </List>
+                <Divider light />
+                <Typography
+                  component="div"
+                  gutterBottom
+                  className={classNames(
+                    classes.cardItem,
+                    classes.cardDescription
+                  )}>
+                  <div>
+                    Assignee: <strong>{item.assignee}</strong>
```

Comment:
Instead of using "Assignee", you can pick an icon fro `mdi-react`:  https://materialdesignicons.com/

For example you can do `import Account from 'mdi-react/AccountIcon'` and use that instead.